src/components/Requirements/CompletedSubReqCourse.vue

```diff
@@ -99,6 +102,20 @@ export default defineComponent({
         } else deleteCourseFromSemesters(this.courseTaken.uniqueId, this.$gtag);
       }
     },
+    openSlotMenu(e: MouseEvent) {
+      console.log('window', window);
+      this.position =
+        window.innerWidth > 863
```

Comment:
Is this what determines if the menu should go left or right? If so, could you make a notion item to apply this same code to our edit course modal? Right now the code that determines which way it should go doesn't work very well